RecoEcal/EgammaClusterProducers/python/SCEnergyCorrectorDRNProducer_cfi.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+DRNProducerEB = cms.EDProducer('SCEnergyCorrectorDRNProducer',
+   correctorCfg = cms.PSet(
+     ecalRecHitsEE = cms.InputTag('reducedEcalRecHitsEE'),
+     ecalRecHitsEB = cms.InputTag('reducedEcalRecHitsEB'),
+     rhoFastJet = cms.InputTag("fixedGridRhoAll"),
+   ),
+   inputSCs = cms.InputTag('particleFlowSuperClusterECAL','particleFlowSuperClusterECALBarrel'),
```

Comment:
Aren't these already specified in fillDescriptions? You should only define config values in python if you want to override something.